packages/react/src/feature-flags.js

```diff
@@ -11,4 +11,5 @@ FeatureFlags.merge({
   'enable-css-custom-properties': true,
   'enable-css-grid': true,
   'enable-v11-release': true,
+  'enable-experimental-tile-contrast': true,
```

Comment:
This needs to be `false` by default, right?

Reply:
Yep, this was just for testing the styles.